docs/user-guide/ui.md

```diff
@@ -118,89 +108,54 @@ track=stable
 
-As an alternative to specifying application details in the deploy wizard, you can define your Replication Controllers and Services in YAML or JSON files, and upload the files to your Pods:
+As an alternative to specifying application details in the deploy wizard, you can define your application in YAML or JSON files, and upload the files using Dashboard:
 
 ![Deploy wizard file upload](/images/docs/ui-dashboard-deploy-file.png)
 
-## Managing resources
-
-### List view
-
-As soon as applications are running on your cluster, Dashboard's initial view defaults to showing all resources available in all namespaces in a list view, for example:
-
-![Workloads view](/images/docs/ui-dashboard-workloadview.png)
-
-For every resource, the list view shows the following information:
-
-* Name of the resource
-* All labels assigned to the resource
-* Number of pods assigned to the resource
-* Age, i.e. amount of time passed since the resource has been created
-* Docker container image
-
-To filter the resources and only show those of a specific namespace, select it from the dropdown list in the right corner of the title bar:
-
-![Namespace selector](/images/docs/ui-dashboard-namespace.png)
-
-### Details view
-
-When clicking a resource, the details view is opened, for example:
+## Using Dashboard
+Following sections describe views of the Kubernetes Dashboard UI; what they provide and how can they be used.
 
-![Details view](/images/docs/ui-dashboard-detailsview.png)
+### Navigation
 
-The **OVERVIEW** tab shows the actual resource details as well as the Pods the resource is running in.
+When there are Kubernetes objects defined in the cluster, Dashboard shows them in the initial view. By default only objects from the _default_ namespace are shown and this can be changed using the namespace selector located in the navigation menu.
 
-The **EVENTS** tab can be useful for debugging applications.
+Dashboard shows most Kubernetes object kinds and groups them in a few menu categories.
 
-To go back to the workloads overview, click the Kubernetes logo.
+#### Admin
+View for cluster and namespace administrators. It lists Nodes, Namespaces and Persistent Volumes and has detail views for them. Node list view contains CPU and memory usage metrics aggregated across all Nodes. The details view shows the metrics for a Node, its specification, status, allocated resources, events and pods running on the node.
 
-### Workload categories
+![Node detail view](/images/docs/ui-dashboard-node.png)
 
-Workloads are categorized as follows:
+#### Workloads
+Entry point view that shows all applications running in the selected namespace. The view lists applications by workload kind (e.g., Deployments, Replica Sets, Stateful Sets, etc.) and each workload kind can be viewed separately. The lists summarize actionable information about the workloads, such as the number of ready pods for a Replica Set or current memory usage for a Pod.
```

Comment:
I've definitely seen PersistentVolume and PersistentVolumeClaim (though usually abbreviated to PVC).